x/txreward/client/cli/tx.go

```diff
@@ -0,0 +1,53 @@
+package cli
+
+import (
+	"github.com/commercionetwork/commercionetwork/x/txreward/internal/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/spf13/cobra"
+)
+
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "CommercioTXREWARDS transactions subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	txCmd.AddCommand(GetCmdIncrementBlockRewardsPool(cdc))
+
+	return txCmd
+}
+
+func GetCmdIncrementBlockRewardsPool(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "increment-rewards-pool [coin-denom] [amount]",
+		Short: "Increments the block rewards pool's liquidity by the given amount",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			txBldr := auth.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
+
+			funder := types.Funder{Address: cliCtx.GetFromAddress()}
+			coin, _ := sdk.NewIntFromString(args[1])
+			amount := sdk.Coin{Denom: args[0], Amount: coin}
```

Comment:
I think the whole code above can be simplified like this

```suggestion
			amount, err := sdk.ParseCoins(args[0])
			if err != nil {
			    return err
			}
```